deepola/experiment.sh

```diff
@@ -44,5 +45,10 @@ do
             query q${qdx} ${scale} \
             ${data_dir}/scale=${scale}/partition=${partition}/${format} ${query_output_dir} ${expt} 2>>$expt_output_dir/results.csv
         set +x
+
+    if [ $j -ne 0 ]; then
```

Comment:
LGTM.
Just a minor thing - the accuracy calculation here is redundant right? For each latency run, we only need the sequence of time-steps. Extract accuracy needs to run only on accuracy run(=0)?
It's fine though if this makes it easier to generate the result format.

Reply:
Oh yes, that's true. The original Q3 and Q10 (other queries with large results?) could be the bottleneck but otherwise should be quick. If we want to plot Q3 and Q10, I can patch this redundancy somehow.